app/controllers/sitcoms_controller.rb

```diff
@@ -22,7 +22,7 @@ def create
   end
 
   def show
-    @reviews = @sitcom.reviews
+    @reviews = @sitcom.reviews.order(created_at: :desc).page params[:page]
```

Comment:
seeing as you are creating a `.order(created_at: :desc)` active record query call here, and in reviews_controller.rb on line 5 could you refactor a 'scope' on the Review model that you could cal in both places? (You should still be able to chain the .page call after that scope).